openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -232,98 +243,87 @@
   implementation_status: not applicable
   narrative:
     - text: |
-        Organizational analysis and correlation of audit records across
-        different repositories to gain organization-wide
-        situational awareness is outside the scope
-        of OpenShift configuration.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
 
 - control_key: AU-6 (4)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
     - text: |
-        Organizational capability to centrally review and analyze
-        audit records from multiple components within the system
-        is outside the scope of OpenShift configuration.
-
-        Note: When AU-4 (1) is enforced OpenShift will be configured
-        to off-load audit records onto a different system
-        or media than the system being audited. In practice this correlates
-        to offloading audit records to a SIEM (e.g. Elastic). However,
-        this control is specifically about centralized review and analysis,
-        and not offloading records, thus AU-6 (4) is marked as not
-        applicable.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
```

Comment:
The suggest text does not seem to align with the NIST requirement language at https://nvd.nist.gov/800-53/Rev4/control/Au-6#enhancement-4



AU-6(4) | AUDIT REVIEW, ANALYSIS, AND REPORTING \| CENTRAL REVIEW AND ANALYSISThe information system provides the capability to centrally review and analyze audit records from multiple components within the system.
-- | --


Can you elaborate on why the existing language needs to be changed?